src/config.ts

```diff
@@ -0,0 +1,245 @@
+"use strict";
```

Comment:
The logic in this file is 100% unmodified besides the addition of a new `get src()` function which does a cast.

It seems GitHub couldn't recognize the TS-ification as a clean diff.

Reply:
Certainly not required, but it would be great to modernize this file a bit as well. Theres a lot of old conventions in here that could be switched easily - private methods that start with __, lots of lodash that can be switched for native methods, etc.

Reply:
I was planning on doing this in a follow-up so that the diff is clear. This is a super critical file and I don't want to mess with it any more than I need to all at once.